pipelines/internal/blob-signing-pipeline/blob-signing-pipeline.yaml

```diff
@@ -38,6 +38,7 @@ spec:
       value: "$(tasks.request-signature.results.signed_payload)"
   tasks:
     - name: collect-simple-signing-params
+      retries: 5
```

Comment:
Just curious, is there a reason why we set 5 retries in some places and in others 3?

Reply:
In another similar simple-signing-pipelines, it set 5 retries.   I think maybe signing tasks are more unstable.

Reply:
Ah okay makes sense, thanks!